docs/patches.md

```
@@ -0,0 +1,38 @@
+# Patch management
+
+Patches apply in **alphabetical filename order**. Numbers in filenames control application order.
+
+## Lifecycle
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)